main.go

```diff
@@ -201,6 +202,16 @@ func main() {
 			return
 		}
 
+		if cfg.auth.Authentication.Header.Enabled {
+			forwardedFor, err := getOutboundIP(upstreamURL.Host)
+			if err != nil {
+				glog.Errorf("Failed to find outbound IP for upstream %s: %v", upstreamURL.Host, err)
+				http.Error(w, "Service currenlty unavailable", http.StatusServiceUnavailable)
```

Comment:
please use the constant text instead: https://github.com/golang/go/blob/go1.11.5/src/net/http/status.go#L134

Reply:
there is also a helper lookup method in https://github.com/golang/go/blob/go1.11.5/src/net/http/status.go#L146